src/features/operator/Market/MarketWithdraw/index.tsx

```diff
@@ -118,6 +132,126 @@ export const MarketWithdraw = (): JSX.Element => {
     }
   };
 
+  const baseToPreferredCurrency = React.useMemo(() => {
```

Comment:
Instead of having the exact same code duplicated twice in useMemo for base and quote, better to have function `convertAmountToFavCurrency(amount: number)`, eventually in a useCallback. `useMemo` doesn't seem the right approach here (https://blog.logrocket.com/rethinking-hooks-memoization). You can set the result of `convertAmountToFavCurrency()` in a `useState` variable and use it in the main JSX block.

Reply:
Working on it.